target/linux/generic/pending-6.12/920-mangle_bootargs.patch

```diff
@@ -13,9 +13,9 @@ Signed-off-by: Imre Kaloz <kaloz@openwrt.org>
 
 --- a/init/Kconfig
 +++ b/init/Kconfig
-@@ -1820,6 +1820,15 @@ config DEBUG_RSEQ
- 
- 	  If unsure, say N.
+@@ -1879,6 +1879,15 @@ config ARCH_HAS_MEMBARRIER_CALLBACKS
+ config ARCH_HAS_MEMBARRIER_SYNC_CORE
+ 	bool
  
 +config MANGLE_BOOTARGS
 +	bool "Rename offending bootargs"
```

Comment:
Move this block down to sort alphabetically.

Reply:
The order is the same as for 6.6.
What exactly should I move and where?